src/app/shared/util/date.util.ts

```diff
@@ -264,7 +302,55 @@ export const getAmountOfIntervals = (beginDate: Date, endDate: Date, interval: I
       return 1 + (duration / (1000 * 60 * 60));
     case INTERVAL_DAILY:
       return 1 + (duration / (1000 * 60 * 60 * 24));
+    case INTERVAL_WEEKLY:
+      return 1 + (duration / (1000 * 60 * 60 * 24 * 7));
+    case INTERVAL_MONTHLY:
+      return 1 + (12 * endDate.getUTCFullYear() - 12 * beginDate.getUTCFullYear()) + (endDate.getUTCMonth() - beginDate.getUTCMonth());
```

Comment:
So these are just used by GA then?  (Attempting to figure out where/how this is used, since monthly is non-consistent one).

Reply:
yep, just used for the value on google analytics events, which isn't so easy to drill down into for individual events anyway, so not sure how useful it even is